web/react/components/delete_post_modal.jsx

```diff
@@ -149,10 +157,10 @@ export default class DeletePostModal extends React.Component {
                         {'Cancel'}
                     </button>
                     <button
+                        ref='deletePostBtn'
                         type='button'
                         className='btn btn-danger'
                         onClick={this.handleDelete}
-                        autoFocus='autofocus'
```

Comment:
I don't think that would work. The problem with this is that when the EditPostModal is closed, it passes focus back to whatever opened it in the first place because we use Bootstrap's `data-toggle` to open it (see [here](https://github.com/twbs/bootstrap/blob/880a3c1a86d6c936624a745954a69febb9e9c354/js/modal.js#L333) where $this is the button that opens the modal and $target is the modal itself).

I fixed something like this a while ago with one of the new channel modals, but I can't remember exactly what I did. I'll see if I can find the PR.